kitchen.yml

```diff
@@ -10,10 +10,10 @@ provisioner:
   product_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
   install_strategy: always
   log_level: <%= ENV['CHEF_LOG_LEVEL'] || 'auto' %>
+  multiple_converge: 2
```

Comment:
Should we also add `enforce_idempotency: true` too?

Reply:
It was primarily put into test the load current value code didn't barf but I can't see why not as those are one and the same in this case.

Reply:
I've been trying to add that to ensure we don't run into issues down the road.